devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/resteasy-jackson-example/java/src/main/java/org/acme/rest/json/Fruit.java

```diff
@@ -0,0 +1,14 @@
+package org.acme.rest.json;
```

Comment:
I am not sure but maybe better to use `org.acme.rest.resteasyjackson` as package name?

Reply:
Good suggestion! I'll change it.